.coveragerc

```diff
@@ -1,7 +1,6 @@
 [run]
 branch = True
 concurrency = multiprocessing,gevent
-plugins = Cython.Coverage
```

Comment:
Why remove this plugin?

Reply:
Cython.Coverage has a bug that blocks collecting coverage data from *.py (https://github.com/cython/cython/issues/1985)